### PR TITLE
[Merged by Bors] - refactor(SetTheory/Ordinal/Arithmetic): deprecate `blsub₂`

### PR DESCRIPTION
**Pull request by vihdzp**
See the new docstring of `blsub₂` for the reasoning.

---

- [x] depends on: #16996

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Ordinal/FixedPoint.lean [hunk 1]
```diff
@@ -5,6 +5,7 @@ Authors: Violeta Hernández Palacios, Mario Carneiro
 -/
 import Mathlib.SetTheory.Ordinal.Enum
 import Mathlib.SetTheory.Ordinal.Exponential
+import Mathlib.Logic.UnivLE
```
Comment (YaelDillies) [inline Mathlib/SetTheory/Ordinal/FixedPoint.lean:8]: Not sure why you need this import suddenly

**Review by YaelDillies — APPROVED** [review]
maintainer merge

File: Mathlib/SetTheory/Ordinal/FixedPoint.lean (hunk 1 above)
Comment (vihdzp) [reply Mathlib/SetTheory/Ordinal/FixedPoint.lean:8]: That might have been added by mistake, let me check.

**Comment by kim-em** [comment]
bors merge

**Pull request closed without merging**